infra/base-images/base-builder/compile_honggfuzz

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash -eu
+# Copyright 2016 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+echo -n "Compiling honggfuzz to $LIB_FUZZING_ENGINE ..."
+
+pushd $SRC/honggfuzz > /dev/null
+make clean
+CC=clang CFLAGS= make
+# libhfuzz.a willl be added by CC/CXX linker directly during linking,
+# but its defined here to satisfy the build infrastructure
+cp libhfuzz/libhfuzz.a $LIB_FUZZING_ENGINE
+cp honggfuzz hfuzz_cc/hfuzz-clang hfuzz_cc/hfuzz-clang++ $OUT/
```

Comment:
any reason why hfuzz-clang/hfuzz-clang++ needs to be in $OUT instead of doing 

`export CC=$SRC/honggfuzz/hfuzz_cc/hfuzz-clang` ?